publishing/README.md

```diff
@@ -0,0 +1,8 @@
+# Publishing conventions
+
+Your Starcounter app is best enjoyed when it is blendable with other apps. For that it needs to follow some conventions.
```

Comment:
I don't like "best enjoyed" term. What about straight and simple: "Your Starcounter app might be used by others when it's blendable."